src/server/rdb_load.cc

```diff
@@ -1393,10 +1392,24 @@ error_code RdbLoaderBase::ReadStringObj(RdbVariant* dest) {
     }
   }
 
+  if (big_string_split && len > kMaxStringSize) {
```

Comment:
do we split a compressed string as well?

Reply:
we do not compress string on save, only big blobs f.e f.e entries together

Reply:
hmm we actually do if we use CompressionMode::SINGLE_ENTRY baaaaa

Reply:
so incase the string is compressed the opcode is RDB_ENC_LZF and we dont split.
we dont realy use CompressionMode::SINGLE_ENTRY so I dont think we need to optimise for this